src/lib/components/DisciplineForm.svelte

```
@@ -1,24 +1,27 @@
+	let { challengeId, clubId }: { challengeId: string; clubId: string } = $props();
 
 	let sheetOpen = $state(false);
+	let rows = $state([{ name: '', multiplier: 1 }]);
 
-	const disciplineForm = superForm(formData, {
-		dataType: 'json',
-		onResult: ({ result }) => {
-			if (result.type === 'success') sheetOpen = !sheetOpen;
-		}
-	});
+	const disciplineForm = addDisciplines.for(challengeId);
```

ESLint

[error] This reference only captures the initial value of `challengeId`. Did you mean to reference it inside a closure instead?
https://svelte.dev/e/state_referenced_locally(state_referenced_locally)

(svelte/valid-compile)